packages/react/react/src/components/Portal/index.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react';
+import { Portal } from './Portal';
+
+const TestComponent = ({ id }: { id: string }) => {
+  return (
+    <>
+      <div id="parent">
+        <Portal id={id}>
+          <div id="child">Example Portal</div>
+        </Portal>
+      </div>
+      <div id="outer"></div>
+    </>
+  );
+};
+
+describe('Portal', () => {
+  it("If the ID of a DOM Node added outside of the parent component's DOM hierarchy is the same as the ID added with the Portal Component props, the Portal Component's Children are rendered as children of the external DOM Node.", () => {
+    const { container } = render(<TestComponent id="outer" />);
+
+    const parent = container.querySelector('#parent');
+    const outer = container.querySelector('#outer');
+
+    const parentChildren = parent?.querySelector('#child');
+    const outerChildren = outer?.querySelector('#child');
+
+    expect(parentChildren).not.toBeInTheDocument();
+    expect(outerChildren).toBeInTheDocument();
+  });
+
+  it("If the ID of a DOM Node added outside of the parent component's DOM hierarchy is different from the ID added with the Portal Component props, the Portal Component's Children are rendered as children of the parent component.", () => {
```

Comment:
I think many users who use `Portal` component do now want this feature😄
How about writing this part in the `document` because I think it's a side effect

Reply:
I understood your comment as adding what you mentioned in the ".md" document. 

Am I understanding your comment correctly?!

If this is correct, I will fix this as well!

Thank you for the feedback 🙂🙂

Reply:
yes I think it is good to be in .md file 😄